controllers/certificate_request_controller.go

```diff
@@ -197,17 +197,19 @@ func (r *CertificateRequestReconciler) Reconcile(ctx context.Context, req ctrl.R
 		l.Info("Signing", "CertificateRequest", req)
 		cert, err := ca.Sign(cr.Spec.Request, keyUsage, extKeyUsage)
 		if err != nil {
-			return ctrl.Result{}, fmt.Errorf("failed to sign CSR for signing: %v", err)
+			return ctrl.Result{}, fmt.Errorf("failed to sign CR for signing: %v", err)
```

Comment:
The log message here could be better... maybe "failed to sign the CertificateRequest: %v" or similar. I think CR is bit confusing here because it could mean both CertificateRequest and CustomResource in this context.

Reply:
Fixed.